apps/site/snippets/en/download/fnm.bash

```diff
@@ -1,14 +1,11 @@
-# installs fnm (Fast Node Manager)
-curl -fsSL https://fnm.vercel.app/install | bash
+# Download and install fnm:
+${props.os === 'WIN' ?
+  'winget install Schniz.fnm' :
+  'curl -o- https://fnm.vercel.app/install | bash'
+}
```

Comment:
if this is handling windows, shouldn't there be `fnm.ps1` and `fnm.bash`? the `.bash` extension doesn't really make sense here if it's not restricted to mac/linux

Reply:
Chatted over Slack